mobius-android/src/main/java/com/spotify/mobius/android/MobiusAndroid.java

```diff
@@ -40,4 +42,13 @@ private MobiusAndroid() {
       MobiusLoop.Factory<M, E, F> loopFactory, M defaultModel, Init<M, F> init) {
     return Mobius.controller(loopFactory, defaultModel, init, MainThreadWorkRunner.create());
   }
+
+  @Nonnull
+  public static <V> Observer<Accumulator<V>> unfold(@Nonnull Observer<V> observer) {
```

Comment:
it would seem to me that this should rather be a method on the accumulator? I think it also would be helpful to use a name like `Accumulator.handleAll()`

Reply:
I'm not sure how to fit this inside the Accumulator. 
This is just a bit of utility that might not be necessary for when subscribing to the accumulated live data, so instead of this:
```
viewModelController
  .viewEffectEmitter()
  .observe(
    lifecycleOwner, 
    accumulator -> { accumulator.handle( this::handleViewEffect )}
  )
```
we could do this
```
viewModelController
  .viewEffectEmitter()
  .observe(lifecycleOwner, unfold(this::handleViewEffect)}
```
I can always remove it since it's just utility.